h_periodic/h_beat.py

```diff
@@ -51,6 +52,12 @@
             "schedule": timedelta(minutes=1),
             "kwargs": {"limit": 400},
         },
+        "fill-annotations-pk": {
+            "options": {"expires": 30},
+            "task": "h.tasks.annotations.fill_pk_and_user_id",
+            "schedule": crontab(hour="*,10-14", minute=0),
+            "kwargs": {"batch_size": 1000},
```

Comment:
My intuition is that the select part of the querying, finding the ones to update, will takes most of the times and won't be linear to the size, I reckon that there won't be much difference (or big difference) between a batch of 1000 and one of 10_000. We'll have to run it in prod to  find out exactly.